Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM ubuntu:16.04
 
-RUN apt-get update && apt-get install -y ruby-dev libffi-dev apache2 git build-essential libxml2-dev zlib1g-dev && gem install bundler
+RUN apt-get update && apt-get install -y ruby-dev libffi-dev apache2 git build-essential libxml2-dev zlib1g-dev && gem install bundler -v '~> 1.11'
```

Comment:
The docker image is unable to build with newer version of bundler ([see here](https://travis-ci.org/OSUOSC/website/builds/484141798#L1414))